styles/Home.module.css

```diff
@@ -129,20 +88,99 @@
   box-shadow: 0 0 20px -7px rgba(0, 0, 0, 0.65);
 }
 
-.content {
-  flex-shrink: 1;
+
+
+
+
+.navBar > ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display:flex;
+  background-color: #93D973;
+  align-items:center;
+  font-size: 30px;
+
+  
+}
+.navBar li {
+  display: block;
+  color:#f2f2f2;
+  text-align: center;
+  padding: 0px 16px 0px 0px;
+  text-decoration: none;
+  
+  
 }
 
-@media only screen and (max-width: 600px) {
-  .header {
-    flex-direction: column;
-  }
-  .coins {
-    padding-top: 20px;
-    justify-content: space-evenly;
-    align-self: center;
+
+
+.navBar Link {
+  
+  border: 10px solid
+}
+
+
+.navBar li a:hover {
+ text-decoration: underline;
+}
+
+
+
+	.homeContainer {
+		display: grid;
+    gap: 10px;
```

Comment:
Not properly indented.

Reply:
Great catches @swarajpure

Reply:
@Kratika0907 I was in the thought it gets automatically indented on save. It is a good idea to have that right?